base/abstractarray.jl

```diff
@@ -2757,6 +2756,36 @@ julia> map(+, [1, 2, 3], [10, 20, 30, 400, 5000])
  33
 ```
 """
+map
+
+"""
```

Comment:
You can't document the same thing twice:

��� Warning: Replacing docs for `Base.map :: Union{}` in module `Base`
��� @ Base.Docs docs/Docs.jl:240

Maybe you meant to just join them?
```suggestion

```

Reply:
Oh I didn't notice, sorry. I did think it should be two docstrings, and that the one for generic collections should come first.